### PR TITLE
2018-02-16_User Feedback Public Meeting

### PR DESCRIPTION
**Pull request by mihaiep**

### CODE REVIEW

File: meetings/2018-02-16_UserFeedbackmeeting.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+**Survey Q#9:** What type of optimization passes do you run on your JavaScript you deploy it to Node.Js?
+ 	Answered 211: Minification 68%, Bundling 60%, Obfuscation 20%, Other 22%
+  	
+**Survey Q#16:** Do you have use cases where you cannot use Node.js because it is single threaded?
+ 	Answered 282: Yes 22%, No 78%
+* Nikolay: anything that need to use cpu that’s a bottleneck
+* Joel: a single threat definitely is something that we’re constantly trying to have to work around it, because we do e-commerce and the server can general do one to two requests per second    
+ 
+## Questions, remarks:
+* Michael: we can help with scripts for IBM I platforms/ running on same benchmark.
```
Comment (mhdawson) [inline meetings/2018-02-16_UserFeedbackmeeting.md]: I think it was Aaron who said that.

File: meetings/2018-02-16_UserFeedbackmeeting.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+* Tierney: Should we provide a way to go in benchmark for like user land?
+* Michael: Is something that users can contribute to cover up of our Matrix,
```
Comment (mhdawson) [inline meetings/2018-02-16_UserFeedbackmeeting.md]: Is something that users can contribute to in order to increase coverage in our matrix.

**Comment by mihaiep** [comment]
Thank you for reviewing this.

**Comment by dshaw** [comment]
@mihaiep Great job getting this cleaned up.

**Pull request merged**